versioned_sidebars/version-8.0-sidebars.json

```diff
@@ -291,7 +291,11 @@
         }
       ],
       "Optimize": [
-        "components/optimize/what-is-optimize",
+        {
+          "type": "link",
+          "label": "What is Optimize?",
+          "href": "/optimize/what-is-optimize"
```

Comment:
I think this is why your build is failing - you'll need a redirect from `/components/optimize/what-is-optimize` to `/optimize/what-is-optimize`.

Reply:
yeah...all these moved docs are going to fail on the link-check. I'll do all the redirects together in a future PR, and hopefully we'll see everything go green there!